.github /workflows/rgrace-workflow.yaml

```diff
@@ -0,0 +1,27 @@
+Name: Build and run Dockerfile for rgrace
+
+on:
+  pull_request:
+    branches: [master]
+     - master
```

Comment:
You combined two correct things here, but it's only correct syntax to have one of the two. 

I suggest you remove line 6 here. I suspect the job will run once you push that change.